js/plugins/checks/src/evaluation.ts

```diff
@@ -104,15 +116,14 @@ function createPolicyEvaluator(
   projectId: string,
   auth: GoogleAuth,
   ai: Genkit,
-  policy_config: ChecksEvaluationMetricConfig
+  policy_config: ChecksEvaluationMetricConfig[],
```

Comment:
This still looks like it is defining one evaluator per policy (which will make one API call per policy). 

You'd want to define a single genkit evaluator that accepts an array of policies and evaluates for all of them. The response can include multiple `ScoreSchema` objects.

Reply:
I left the original logic of creating an evaluator instance for each policy and an evaluator for all the policies configured. The thought was that some users might want to just run one of the policies individually.

That happens here in the constructor:
```ts
  // Individual evaluators, one per configured metric.
  const evaluators = policy_configs.map((policy_config) => {
    return createPolicyEvaluator(
      projectId,
      auth,
      ai,
      [policy_config],
      policy_config.type as string
    );
  });

  // Single evaluator instnace with all configured policies.
  evaluators.push(
    createPolicyEvaluator(projectId, auth, ai, policy_configs, 'all_metrics')
  );
```ts
I'm not sure if that's the right approach though. I'm leaning towards just removing all the individual evaluators and having a single evaluator named checks that runs all configured policies every time. If a user wants to run a different set of policies or one in isolation then they would have to update their plugin config. What do you think?

Reply:
@ssbushi removed the extra evaluators. Users can now just run `genkit eval:run test-dataset.json --evaluators=checks`